packages/app/src/update/actions/start-update-os/start-update-os.action.ts

```diff
@@ -62,6 +67,17 @@ export const startUpdateOs = createAsyncThunk<
 
     for (const release of releases) {
       state = getState() as RootState & ReduxRootState
+      const trackOsUpdateOptions: Omit<TrackOsUpdateOptions, "state"> = {
+        deviceType: state.device.deviceType as DeviceType,
```

Comment:
state.device.deviceType might be DeviceType or null. But what if it will be null and you'll cast it to DeviceType? It will be still null. Maybe it could be figured out differently? Like state.device.deviceType ?? DeviceType.MuditaPure?